build.gradle

```diff
@@ -63,7 +64,7 @@ allprojects {
 	}
 	tasks.withType(KotlinCompilationTask.class)
 			.configureEach {
-				compilerOptions.languageVersion = KotlinVersion.KOTLIN_1_7
+				compilerOptions.languageVersion = KotlinVersion.KOTLIN_1_8
```

Comment:
Apparently Kotlin 2.2 doesn't allow language version 1.7 anymore. It also complains that version 1.8 is deprecated and will lose support in the future, but I went conservative here, anyway.